azure-keyvault/src/main/java/com/microsoft/azure/keyvault/messagesecurity/JWEHeader.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.keyvault.messagesecurity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+
+/**
+ * Json Web Encryption Header class.
+ */
+public class JWEHeader {
```

Comment:
>public  [](start = 0, length = 7)

Remove public security modifier.  We might expose this class at a later time in a different package so we should prevent people from taking a dependency on this version by making it only accessible in messagesecurity.